lib/common/options.c

```diff
@@ -691,37 +689,37 @@ pcmk__format_option_metadata(const char *name, const char *desc_short,
             char *ptr = strtok(str, delim);
 
             if (opt_default != NULL) {
-                g_string_append_printf(s, "      "
-                                       "<content type=\"%s\" default=\"%s\">\n",
-                                       opt_type, opt_default);
+                pcmk__g_strcat(s, "      "
+                               "<content type=\"", opt_type, "\" "
+                               "default=\"", opt_default, "\">\n", NULL);
             } else {
-                g_string_append_printf(s, "      "
-                                       "<content type=\"%s\">\n", opt_type);
+                pcmk__g_strcat(s, "      "
+                               "<content type=\"", opt_type, "\">\n", NULL);
             }
```

Comment:
I think this would be easier to follow with

    ... <content type=...
    if (opt_default) != NULL) { ...default=... }
    ...>\n...

and we can avoid the repeated block below by doing that first regardless, then checking for select (just the close tag needs attention)

Reply:
Agreed